tests/unit/test_parameters/test_update_parameter_sets_doc.py

```diff
@@ -0,0 +1,63 @@
+"""
+Tests for the update_parameter_sets_doc.py
+"""
+from pybamm.parameters.update_parameter_sets_doc import generate_ps_doc
+import pybamm
+import unittest
+
+
+DOC = """
+Parameter sets from papers. The 'citation' entry provides a reference to the appropriate
+paper in the file "pybamm/CITATIONS.txt". To see which parameter sets have been used in
+your simulation, add the line "pybamm.print_citations()" to your script.
+
+Lead-acid parameter sets
+------------------------
+    * Sulzer2019 :
+        Valentin Sulzer, S. Jon Chapman, Colin P. Please, David A. Howey, and Charles W.
+        Monroe. Faster Lead-Acid Battery Simulations from Porous-Electrode Theory: Part
+        I. Physical Model. Journal of The Electrochemical Society, 166(12):A2363–A2371,
+        2019. doi:10.1149/2.0301910jes.
+
+Lithium-ion parameter sets
+--------------------------
+    * Ecker2015 :
+        Madeleine Ecker, Thi Kim Dung Tran, Philipp Dechent, Stefan Käbitz, Alexander
+        Warnecke, and Dirk Uwe Sauer. Parameterization of a Physico-Chemical Model of a
+        Lithium-Ion Battery: I. Determination of Parameters. Journal of the
+        Electrochemical Society, 162(9):A1836–A1848, 2015.
+        doi:10.1149/2.0551509jes.Giles Richardson, Ivan Korotkin, Rahifa Ranom, Michael
```

Comment:
need a new line here between refs

Reply:
@tinosulzer Could you please confirm if it's correct now?
```
Parameter sets from papers. The 'citation' entry provides a reference to the appropriate
paper in the file "pybamm/CITATIONS.txt". To see which parameter sets have been used in
your simulation, add the line "pybamm.print_citations()" to your script.

Lead-acid parameter sets
------------------------
    * Sulzer2019 :
        Valentin Sulzer, S. Jon Chapman, Colin P. Please, David A. Howey, and Charles W.
        Monroe. Faster Lead-Acid Battery Simulations from Porous-Electrode Theory: Part
        I. Physical Model. Journal of The Electrochemical Society, 166(12):A2363–A2371,
        2019. doi:10.1149/2.0301910jes.

Lithium-ion parameter sets
--------------------------
    * Ai2020 :
        Weilong Ai, Ludwig Kraft, Johannes Sturm, Andreas Jossen, and Billy Wu.
        Electrochemical thermal-mechanical modelling of stress inhomogeneity in
        lithium-ion pouch cells. Journal of The Electrochemical Society, 167(1):013512,
        2019. doi:10.1149/2.0122001JES.
    * Chen2020 :
        Chang-Hui Chen, Ferran Brosa Planella, Kieran O'Regan, Dominika Gastol, W.
        Dhammika Widanage, and Emma Kendrick. Development of Experimental Techniques for
        Parameterization of Multi-scale Lithium-ion Battery Models. Journal of The
        Electrochemical Society, 167(8):080534, 2020. doi:10.1149/1945-7111/ab9050.
    * Chen2020_plating :
        Chang-Hui Chen, Ferran Brosa Planella, Kieran O'Regan, Dominika Gastol, W.
        Dhammika Widanage, and Emma Kendrick. Development of Experimental Techniques for
        Parameterization of Multi-scale Lithium-ion Battery Models. Journal of The
        Electrochemical Society, 167(8):080534, 2020. doi:10.1149/1945-7111/ab9050.
    * Ecker2015 :
        Madeleine Ecker, Stefan Käbitz, Izaro Laresgoiti, and Dirk Uwe Sauer.
        Parameterization of a Physico-Chemical Model of a Lithium-Ion Battery: II. Model
        Validation. Journal of The Electrochemical Society, 162(9):A1849–A1857, 2015.
        doi:10.1149/2.0541509jes.
        Madeleine Ecker, Thi Kim Dung Tran, Philipp Dechent, Stefan Käbitz, Alexander
        Warnecke, and Dirk Uwe Sauer. Parameterization of a Physico-Chemical Model of a
        Lithium-Ion Battery: I. Determination of Parameters. Journal of the
        Electrochemical Society, 162(9):A1836–A1848, 2015. doi:10.1149/2.0551509jes.
        Giles Richardson, Ivan Korotkin, Rahifa Ranom, Michael Castle, and Jamie M.
        Foster. Generalised single particle models for high-rate operation of graded
        lithium-ion electrodes: systematic derivation and validation. Electrochimica
        Acta, 339:135862, 2020. doi:10.1016/j.electacta.2020.135862.
    * Marquis2019 :
        Scott G. Marquis, Valentin Sulzer, Robert Timms, Colin P. Please, and S. Jon
        Chapman. An asymptotic derivation of a single particle model with electrolyte.
        Journal of The Electrochemical Society, 166(15):A3693–A3706, 2019.
        doi:10.1149/2.0341915jes.
    * Mohtat2020 :
        Peyman Mohtat, Suhak Lee, Valentin Sulzer, Jason B. Siegel, and Anna G.
        Stefanopoulou. Differential Expansion and Voltage Model for Li-ion Batteries at
        Practical Charging Rates. Journal of The Electrochemical Society,
        167(11):110561, 2020. doi:10.1149/1945-7111/aba5d1.
    * NCA_Kim2011 :
        Gi-Heon Kim, Kandler Smith, Kyu-Jin Lee, Shriram Santhanagopalan, and Ahmad
        Pesaran. Multi-domain modeling of lithium-ion batteries encompassing
        multi-physics in varied length scales. Journal of the Electrochemical Society,
        158(8):A955–A969, 2011. doi:10.1149/1.3597614.
    * Prada2013 :
        Chang-Hui Chen, Ferran Brosa Planella, Kieran O'Regan, Dominika Gastol, W.
        Dhammika Widanage, and Emma Kendrick. Development of Experimental Techniques for
        Parameterization of Multi-scale Lithium-ion Battery Models. Journal of The
        Electrochemical Society, 167(8):080534, 2020. doi:10.1149/1945-7111/ab9050.
        Michael J. Lain, James Brandon, and Emma Kendrick. Design strategies for high
        power vs. high energy lithium ion cells. Batteries, 5(4):64, 2019.
        doi:10.3390/batteries5040064.
        Eric Prada, D. Di Domenico, Y. Creff, J. Bernard, Valérie Sauvant-Moynot, and
        François Huet. A simplified electrochemical and thermal aging model of
        LiFePO4-graphite Li-ion batteries: power and capacity fade simulations. Journal
        of The Electrochemical Society, 160(4):A616, 2013. doi:10.1149/2.053304jes.
    * Ramadass2004 :
        Scott G. Marquis, Valentin Sulzer, Robert Timms, Colin P. Please, and S. Jon
        Chapman. An asymptotic derivation of a single particle model with electrolyte.
        Journal of The Electrochemical Society, 166(15):A3693–A3706, 2019.
        doi:10.1149/2.0341915jes.
```

Reply:
Looks better, maybe we should start each individual citation with a `-` or `>` to more clearly separate them.

Reply:
Sounds good, I'll add them